src/renderer/main.js

```diff
@@ -132,21 +123,23 @@ function lazyLoadCast () {
   return Cast
 }
 
-// This is the (mostly) pure function from state -> UI. Returns a virtual DOM
-// tree. Any events, such as button clicks, will turn into calls to dispatch()
-function render (state) {
-  try {
-    return App(state)
-  } catch (e) {
-    console.log('rendering error: %s\n\t%s', e.message, e.stack)
+// Calls render() to go from state -> UI, then applies to vdom to the real DOM.
+// Runs at 60fps, but only executes when necessary
+var needsRender = 0
+
+function renderIfNecessary () {
+  if (needsRender > 1) console.log('combining %d update() calls into one update', needsRender)
```

Comment:
If this log gets printed out often, it will be a perf hit and spam the console. We might want to axe it once we're sure this code is working correctly.